### PR TITLE
Added  Orika mapper & implemented it mapping in all existing operations.

### PR DESCRIPTION
**Pull request by JoseLazcano-JLo**
Included two US in one go:
-US 19 (created configuration class for MappingFactory and DTO models) and US 24 (implemented them into the main flow for the operations Post Login User, Post Create User, Get Users, Get by Id).

### CODE REVIEW

**Review by luisgalindoluna — APPROVED** [review]
Noice

File: src/main/java/com/agilethought/atsceapi/domain/UserDTO.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.agilethought.atsceapi.domain;
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/domain/UserDTO.java]: DTO models should be in other package called dto

File: src/main/java/com/agilethought/atsceapi/domain/UserDTO.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+@Getter
+@Setter
+public class UserDTO {
+	private String id;
+	private Integer type;
+	private String firstName;
+	private String lastName;
+	private String email;
+	private String password;
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/domain/UserDTO.java]: additionally, we need to remove sensitive data from UserDTO, we need to remove password from responses of get.

**Review by ivanflores15 — CHANGES_REQUESTED** [review]

File: src/main/java/com/agilethought/atsceapi/controller/UserController.java [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
-import com.agilethought.atsceapi.model.User;
-import org.springframework.web.bind.annotation.*;
+import com.agilethought.atsceapi.domain.LoginData;
 import com.agilethought.atsceapi.domain.NewUserRequest;
 import com.agilethought.atsceapi.domain.NewUserResponse;
+import com.agilethought.atsceapi.domain.UserDTO;
+import com.agilethought.atsceapi.service.UserService;
 
 @RestController
 @RequestMapping(value = "/at-sce-api/api/v1")
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/controller/UserController.java]: @RequestMapping(value = "/api/v1") should be new path

**Review by ivanflores15 — CHANGES_REQUESTED** [review]

**Review by ivanflores15 — APPROVED** [review]
look good, good job

**Review by EmiliaGuillen-dev — APPROVED** [review]
looks good

**Pull request merged**